README.md

```diff
@@ -0,0 +1,40 @@
+# Neutron Logger
+
+This repository contains a helper zap.Logger constructor that injects a mandatory "context" field to the logger fields. The context field is expected to carry information about the execution context the logger is used in to ease logs browsing and indexing.
+
+The logging level is set using an env variable `LOG_LEVEL`. According to the zap.Logger docs, the expected `LOG_LEVEL` values are (the `dpanic`/`DPANIC` level is omitted, higher levels are more important):
+- `debug` or `DEBUG` — typically voluminous logs that are usually disabled in production;
+- `info` or `INFO` — the default logging priority;
+- `warn` or `WARN` — logs that are more important than Info, but don't need individual human review;
+- `error` or `ERROR` — high-priority logs. If an application is running smoothly, it shouldn't generate any error-level logs;
+- `panic` or `PANIC` — logs a message, then panics;
+- `fatal` or `FATAL` — logs a message, then calls os.Exit(1).
+
+### Example
+
+```go
+package main
+
+func main() {
+	l, err := NewForContext("my_application")
+	if err != nil {
+		panic(err)
+	}
+	l.Debug("debug")
+	l.Info("info")
+	l.Warn("warm")
+	l.Error("error")
+	l.Fatal("fatal")
+}
+```
+
+`.env` file:
+```
+LOG_LEVEL=error
+```
+
+results in (stack trace message is omitted):
+```
+{"level":"error","ts":1663859412.242798,"caller":"playground/main.go:11","msg":"error","context":"my_application"}
+{"level":"fatal","ts":1663859412.242861,"caller":"playground/main.go:12","msg":"fatal","context":"my_application"}
```

Comment:
The caller doesn't look any different compared to what is described in Jira task. I mean, it would still not be great for e.g. Elastic Search. Is it okay for us or is this task not finished yet?

Reply:
the `thing` is not in the caller, but in the `context` field. the context field is set manually at logger creation as a parameter, and then when browsing logs this value is used to filter logs out

Reply:
alright, so, this example doesn't show it completely. Aside from it, everything seems to be alright for me.